python/docs/examples/bluesky.md

```diff
@@ -12,13 +17,7 @@ You can see the full example in the file [bluesky.py](https://github.com/kaskada
 Before we can receive events from Bluesky, we need to create a data source to tell Kaskada how to handle the events.
 We'll provide a schema and configure the time and entity fields.
 
-```{literalinclude} bluesky.py
-:language: python
-:start-after: "[start_setup]"
-:end-before: "[end_setup]"
-:linenos:
-:lineno-match:
-:dedent: 4
+```{.python include="bluesky.py" code-line-numbers="true" start-line=26 end-line=53 dedent=4}
```

Comment:
The fact we need to switch to line numbers isn't great. Maybe worth a feature request and/or PR to the plugin to provide something similar to the tags we used. Those are nice because (a) we can see in the file which parts are included and (b) as we make changes it doesn't break the lines we've chosen to include.